Dockerfile.old

```diff
@@ -0,0 +1,40 @@
+FROM node:16 AS builder
```

Comment:
Please delete this file .... dont need this for now

Reply:
Deleted the old docker file